README.md

```diff
@@ -42,6 +42,11 @@ If you are using Maven without Bom, Add this to your dependencies.
 ```
 If you are using Gradle, add this to your dependencies
 ```Groovy
+repositories {
+        google() //this is required from version 1.104 onwards
+        //other repositories
```

Comment:
Do you need to add `central()` explicitly too?

Reply:
Yes. I added the additional part that would be required

Reply:
you need to put central here too

Reply:
done

Reply:
@elharo should I make any more changes?